lib/elixir/test/elixir/calendar/date_test.exs

```diff
@@ -48,6 +51,12 @@ defmodule DateTest do
     assert inspect(date) == "~D[1/1/2000 FakeCalendar]"
   end
 
+  test "inspect/1 roundtrip to to_string/1" do
+    test_string = "~D[2000-01-01 Calendar.Holocene]"
+    {date, []} = Code.eval_string(test_string)
+    assert inspect(date) == test_string
+  end
```

Comment:
Instead of adding new examples to the suite, let's just change all of the Holocene examples to include 5 digits. What do you think?

Reply:
Let me know if this should still come out? I adjusted all the `Calendar.Holocene` tests to use `years > 9999` but left this since I think its the only deliberately round trip example.

Reply:
The round trip is a property of the calendar but  we are not interested in testing the calendar in this example, so we are fine without it. :)